bauble_macros/src/lib.rs

```diff
@@ -31,6 +35,7 @@ use proc_macro::TokenStream;
 /// - `validate`: Adds extra validation to this type or variant. Expects a function
 ///   like `fn(&Val, &TypeRegistry) -> Result<(), ConversionError>`. Can either
 ///   be a defined function, or a closure.
+/// - `value_default`: Used to give the bauble type a default.
```

Comment:
No docs for `default` attribute.

Reply:
It says `value_default` there as well though. Is that intended?

Reply:
Yeah, it can be specified for fields.